rules/__tests__/consistent-test-it.test.js

```diff
@@ -0,0 +1,352 @@
+        code: 'describe("suite",() => { it.only("foo"); })',
+        options: [{ fn: 'it', withinDescribe: 'test' }],
+        errors: [
+          { message: "Prefer using 'test' instead of 'it' within describe" },
+        ],
+        output: 'describe("suite",() => { test.only("foo"); })',
+      },
+      {
+        code: 'describe("suite",() => { xit("foo"); })',
+        options: [{ fn: 'it', withinDescribe: 'test' }],
+        errors: [
+          { message: "Prefer using 'test' instead of 'it' within describe" },
+        ],
+        output: 'describe("suite",() => { xtest("foo"); })',
+      },
+      {
+        code: 'describe("suite",() => { it.skip("foo"); })',
+        options: [{ fn: 'it', withinDescribe: 'test' }],
+        errors: [
+          { message: "Prefer using 'test' instead of 'it' within describe" },
+        ],
+        output: 'describe("suite",() => { test.skip("foo"); })',
+      },
+    ],
+  }
+);
+
+ruleTester.run(
+  'consistent-test-it with fn=test and withinDescribe=test ',
+  rules['consistent-test-it'],
+  {
+    valid: [
+      {
+        code: 'describe("suite", () => { test("foo")})',
+        options: [{ fn: 'test', withinDescribe: 'test' }],
+      },
+      {
+        code: 'test("foo")',
+        options: [{ fn: 'test', withinDescribe: 'test' }],
+      },
+    ],
+    invalid: [
+      {
+        code: 'describe("suite", () => { it("foo")})',
+        options: [{ fn: 'test', withinDescribe: 'test' }],
+        errors: [
+          { message: "Prefer using 'test' instead of 'it' within describe" },
+        ],
+        output: 'describe("suite", () => { test("foo")})',
+      },
+      {
+        code: 'it("foo")',
+        options: [{ fn: 'test', withinDescribe: 'test' }],
+        errors: [{ message: "Prefer using 'test' instead of 'it'" }],
+        output: 'test("foo")',
+      },
+    ],
+  }
+);
+
+ruleTester.run(
+  'consistent-test-it with fn=it and withinDescribe=it ',
+  rules['consistent-test-it'],
+  {
+    valid: [
+      {
+        code: 'describe("suite", () => { it("foo")})',
+        options: [{ fn: 'it', withinDescribe: 'it' }],
+      },
+      {
+        code: 'it("foo")',
+        options: [{ fn: 'it', withinDescribe: 'it' }],
+      },
+    ],
+    invalid: [
+      {
+        code: 'describe("suite", () => { test("foo")})',
+        options: [{ fn: 'it', withinDescribe: 'it' }],
+        errors: [
+          { message: "Prefer using 'it' instead of 'test' within describe" },
+        ],
+        output: 'describe("suite", () => { it("foo")})',
+      },
+      {
+        code: 'test("foo")',
+        options: [{ fn: 'it', withinDescribe: 'it' }],
+        errors: [{ message: "Prefer using 'it' instead of 'test'" }],
+        output: 'it("foo")',
+      },
+    ],
+  }
+);
+
+ruleTester.run(
+  'consistent-test-it default behavior',
+  rules['consistent-test-it'],
+  {
+    valid: [
+      {
+        code: 'test("foo")',
+        options: [{ fn: 'test', withinDescribe: 'it' }],
```

Comment:
I think removing `options` here is enough to get to 100% coverage 🙂

Reply:
Yes, you are right :) will push fix soon